frontend/src/scenes/experiments/ExperimentNext.tsx

```diff
@@ -0,0 +1,210 @@
+import './Experiment.scss'
+
+import { IconPlusSmall, IconTrash } from '@posthog/icons'
+import { LemonDivider, LemonInput, LemonTextArea, Tooltip } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form, Group } from 'kea-forms'
+import { ExperimentVariantNumber } from 'lib/components/SeriesGlyph'
+import { MAX_VARIANTS } from 'lib/constants'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonField } from 'lib/lemon-ui/LemonField'
+import React, { useEffect } from 'react'
+
+import { experimentLogic } from './experimentLogic'
+
+const CaretRightIcon = ({ fontSize }: { fontSize: number }): JSX.Element => {
```

Comment:
All icons go in icons.tsx :) - check if this already exists there

Reply:
No it doesn't, but will add this icon there 👍

Reply:
Right, we do have this icon already 😅